omero/sysadmins/unix/server-centos7-ice36.rst

```diff
@@ -119,6 +119,15 @@ Configure the database and the location of the data directory:
     :start-after: #end-copy-omeroscript
     :end-before: #end-step04
 
+Weaker ciphers like ADH are disabled by default in new version of OpenSSL
+and TLS versions 1.0 and 1.1 have been dropped from JDK packages.
+This means that it is not possible to connect to an OMERO.server
+using any OMERO clients e.g. the Java Desktop client,
+the OMERO.web client or the CLI and import data. Run:
```

Comment:
This might need rephrasing as this reads as it is still not possible to connect. Maybe `In order to connect,```

```
In order to connect to an OMERO.server using any OMERO clients e.g. the Java Desktop client, the OMERO.web client or the CLI and import data, you need to generate self-signed certificates by running:
```